### ISSUE TITLE
Portable NvDA: object navigation (command or code) does not work when focused on a modern app with compact overlay enabled

### ISSUE THREAD

**Issue by josephsl** [nvaccess/nvda#12069]
Hi,

Same setup as #12068, although it will work better with Calculator:

### Steps to reproduce:
Following the steps outlined in #12068 except for opening Python Console:

1. Open Calculator on Windows 10.
2. In standard calculator mode, perform object navigation from the keyboard (NVDA+Numpad arrows in desktop layout).
3. Press Alt+Up arrow to switch to always on mode.
4. Try performing object navigation to move up and down the tree.

### Actual behavior:
An error tone is heard (because UIA notification handler cannot complete due to object navigation code/IndexError):

ERROR - eventHandler.executeEvent (12:48:40.635) - MainThread (13940):
error executing event: UIA_notification on <NVDAObjects.Dynamic_EditableTextWithAutoSelectDetectionUIA object at 0x06A307A8> with extra args of {'notificationKind': 2, 'notificationProcessing': 1, 'displayString': 'Calculator always on top', 'activityId': 'AlwaysOnTop'}
Traceback (most recent call last):
  File "appModules\calculator.py", line 70, in event_UIA_notification
    obj.previous.UIAAutomationId in
AttributeError: 'NoneType' object has no attribute 'UIAAutomationId'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "eventHandler.py", line 246, in executeEvent
    _EventExecuter(eventName,obj,kwargs)
  File "eventHandler.py", line 96, in __init__
    self.next()
  File "eventHandler.py", line 105, in next
    return func(*args, **self.kwargs)
  File "appModules\calculator.py", line 74, in event_UIA_notification
    resultElement = api.getForegroundObject().children[1].lastChild
IndexError: list index out of range

### Expected behavior:
No errors, object navigation (commands or code) works.

#### NVDA installed/portable/running from source:
Source

#### NVDA version:
master:e5d62e3296 with wxPython 4.1.1

#### Windows version:
Windows 10 Version 20H2

#### Name and version of other software in use when reproducing the issue:
Calculator 10.2101.10.0

#### Other information about your system:
Python 3.8 work, part of Windows Insider beta channel

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No issues in alpha-21753,e5d62e32, 2020.3, 2020.4 RC1 when used as installed version. Issue occurred in portable versions.

#### If addons are disabled, is your problem still occurring?
Yes

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
Not applicable

Thanks.

**Comment by josephsl** [comment]
Hi,

Turns out this is for all portable copies I tested so far (including source code version; updated title and initial comment).

Thanks.

**Comment by Adriani90** [comment]
@josephsl how is this now with NVDA 2024.1 RC1?

**Comment by josephsl** [comment]
Hi,

Let's close this as I can't reproduce this and UIA notification event handler is indeed working in portable cpies. Thanks.
